### ISSUE TITLE
Problem when reproducing the result

### ISSUE THREAD

**Issue by ngoductuanlhp** [aim-uofa/DyCo3D#9]
Thank you for sharing the code. When I re-trained the model on the training set of ScanNetv2 (masking head size of 16) with your default config then tested it on the validation set, the result is lower than the result of your provided pretrained model (see image below).
![dyco3d_retrain_iter36500](https://user-images.githubusercontent.com/39947254/133034214-d54010ba-fa6d-4a7a-a4bd-6fdb7b565792.png)

So I have some concerns about the implementation details:

1. In the paper, the cluster radius is set to 0.03 but in your code is 0.02.
2. In the paper, you used the dice loss to train the semantic segmentation but I have not seen it in your code.
3. I trained the model with batch_size = 6 on a single V100 GPU. Does the batch_size strongly affect the performance?

Could you give me some advices？ Thanks.

**Comment by tonghe90** [comment]
Hi @ngoductuanlhp:
1. the performance reported in the ablation studies is using a radius of 0.03, we found changing it to 0.02 will slightly boost the performance, which is report as the final results.
2. the dice loss we are using is:
def dice_coefficient(x, target):
    eps = 1e-5
    n_inst = x.size(0)
    x = x.reshape(n_inst, -1)
    target = target.reshape(n_inst, -1)
    intersection = (x * target).sum(dim=1)
    union = (x ** 2.0).sum(dim=1) + (target ** 2.0).sum(dim=1) + eps
    loss = 1. - (2 * intersection / union)
    return loss
we will check the training code and add it.
3. As the performance is depending on the results of semantic segmentation, I suggest you train the task of semantic segmentation with a longer iteration, or you can finetune from a model which has already done the semantic segmentation task.

**Comment by ngoductuanlhp** [comment]
Thank you for your response.
